automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPyKernelSpec.scala

```diff
@@ -38,76 +41,9 @@ class NotebookPyKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
-      contentSecurityHeader shouldBe defined
-      contentSecurityHeader.get.value should include("https://bvdp-saturn-dev.appspot.com")
-      contentSecurityHeader.get.value should not include "https://bvdp-saturn-prod.appspot.com"
-      contentSecurityHeader.get.value should not include "*.terra.bio"
-    }
-
-    "should allow BigQuerying via the command line" in { runtimeFixture =>
-      withWebDriver { implicit driver =>
-        withNewNotebook(runtimeFixture.runtime) { notebookPage =>
-          val query =
-            """! bq query --format=json "SELECT COUNT(*) AS scullion_count FROM [bigquery-public-data.samples.shakespeare] WHERE word='scullion'" """
-          val expectedResult = """[{"scullion_count":"2"}]""".stripMargin
-
-          val result = notebookPage.executeCell(query, timeout = 5.minutes).get
-          result should include(expectedResult)
-        }
-      }
-    }
+    // TODO: [discuss] it does not seem like automation tests should be verifying security policies
 
-    "should update dateAccessed if the notebook is open" in { runtimeFixture =>
-      withWebDriver { implicit driver =>
-        withNewNotebook(runtimeFixture.runtime) { _ =>
-          val firstApiCall =
-            Leonardo.cluster.getRuntime(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
-          // Sleeping for 90s to simulate idle notebook
-          logger.info("Sleeping for 120s to simulate idle notebook")
-          Thread.sleep(120000)
-          val secondApiCall =
-            Leonardo.cluster.getRuntime(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName)
-          firstApiCall.auditInfo.dateAccessed should be < secondApiCall.auditInfo.dateAccessed
-        }
-      }
-    }
-
-    s"should have the workspace-related environment variables set in ${Python3.toString} kernel" in { runtimeFixture =>
-      withWebDriver { implicit driver =>
-        val expectedEVs =
-          RuntimeFixtureSpec.getCustomEnvironmentVariables ++
-            // variables implicitly set by Leo
-            Map(
-              "CLUSTER_NAME" -> runtimeFixture.runtime.clusterName.asString,
-              "RUNTIME_NAME" -> runtimeFixture.runtime.clusterName.asString,
-              "OWNER_EMAIL" -> runtimeFixture.runtime.creator.value,
-              // TODO: remove when PPW is rolled out to all workspaces
-              // and Leo removes the kernel_bootstrap logic.
-              // See https://broadworkbench.atlassian.net/browse/IA-2936
-              "WORKSPACE_NAME" -> "home"
-            )
-        withNewNotebook(runtimeFixture.runtime, Python3) { notebookPage =>
-          notebookPage.executeCell("import os")
-          expectedEVs.foreach { case (k, v) =>
-            val res = notebookPage.executeCell(s"os.getenv('$k')")
-            res shouldBe defined
-            res.get shouldBe s"'$v'"
-          }
-        }
-      }
-    }
-
-    // https://github.com/DataBiosphere/leonardo/issues/891
```

Comment:
So I actually caught a bug thanks to this test over the summer. And that is something that we might need to test via leo, in addition to the docker image, because of the dependency on the C bindings in the VM directly

Reply:
Could you move this test back in?